wellregistry/registry/tests/lookups.py

```diff
@@ -0,0 +1,49 @@
+"""
+Module for Managing lookups used in tests.
+"""
+
+from ..models import AgencyLookup, AltitudeDatumLookup, CountryLookup, CountyLookup, HorizontalDatumLookup
+from ..models import NatAqfrLookup, StateLookup, UnitsLookup
+
+def create_units_dim(unit_id, unit_desc):
```

Comment:
Nit: This function seems unnecessary as you can just call UnitsLookup.object.create twice.